tdisplays3-displaypages.yaml

```diff
@@ -0,0 +1,296 @@
+esphome:
+  name: "tdisplays3-displaypages"
+  friendly_name: "tdisplays3-displaypages"
+
+  on_boot: 
+    - priority: 600
+      then:
+        - lambda: |-
+            //id(disp).tft.init();
+            // Set the rotation here, not on the component
+            id(disp).tft.setRotation(3);
+
+            // Fetch driver details from TFT_eSPI and push into ESPHome sensors
+            setup_t tftSetup;
+            id(disp).tft.getSetup(tftSetup);            
+            id(TFT_eSPI_Version).publish_state(tftSetup.version.c_str());
+            int iTFTDriver = tftSetup.tft_driver;
+            id(TFT_eSPI_Driver).publish_state(((String)iTFTDriver).c_str());
+            if(tftSetup.serial == 1) {
+              id(TFT_eSPI_Interface).publish_state("Serial");
+            }
+            else {
+              id(TFT_eSPI_Interface).publish_state("8-bit Parallel");
+            }
```

Comment:
Is this always needed, or could it also be moved to the component's `setup` method?

Reply:
I did this so I could push info into sensor for debugging and just left in in my sample yaml for reference.